README.md

```diff
@@ -90,13 +150,47 @@ To create a new adaptor follow the steps below
 
   [Here's an example of a configuration schema](packages/template/configuration-schema.json)
 
-- To add or update functions to an existing adaptor:
 
-  - Create or update adaptor functions.
+  #### 4. Test it manually 
+  - Make sure to run build again
+  - Run your job `openfn tmp/job.js -O -ma youradaptorname`
+  - Make sure it returns what you expect
+  - You can delete this folder once you have finished testing your job manually
```

Comment:
Which folder? :thinking: 

If you mean `tmp`, then yes technically it can be removed but it's usually useful to keep the job code for later.

Reply:
Ah this because folks often keep secrets in a tmp folder. I think the pattern we want to encourage is to make sure that people keep their local convenience folder (like tmp) _out_ of git.

Reply:
Yes, keeping sensitive stuff out of git is an even better pattern. We could consider re-writing all this to use a path like `~/openfn/jobs/adaptorname` or something.

Another thought - these are very unixy paths and commands we're using - are we alienating windows users?